libs/simulation-runs/ui/src/lib/render-viz/render-viz.component.html

```diff
@@ -1,14 +1,13 @@
-<div class="container">
+<!--<div class="container">
   <div *ngIf="visualization; else noPlot">
     <div *ngIf="projectTitle">
       <div *ngIf="plotTitle">
-        <!-- Case: Vega -->
+
         <biosimulations-vega-visualization
           *ngIf="visualization.renderer === 'Vega'"
           [spec]="visualization.vegaSpec"
           [name]="visualization.name + '_vega'">
```

Comment:
remove commented-out code unless you really need it.

Reply:
Works for me!